src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichelMHD.cpp

```diff
@@ -0,0 +1,382 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichelMHD.hpp"
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"                   // IWYU pragma: keep
+#include "NumericalAlgorithms/RootFinding/TOMS748.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+/// \cond
+namespace grmhd {
+namespace Solutions {
+
+BondiMichelMHD::BondiMichelMHD(
+    const Mass::type mass, const SonicRadius::type sonic_radius,
+    const SonicDensity::type sonic_density,
+    const PolytropicExponent::type polytropic_exponent,
+    const MagFieldStrength::type mag_field_strength) noexcept
+    : mass_(mass),
+      sonic_radius_(sonic_radius),
+      sonic_density_(sonic_density),
+      polytropic_exponent_(polytropic_exponent),
+      mag_field_strength_(mag_field_strength),
+      // From Rezzola and Zanotti (2013)
```

Comment:
I'd prefer you do `Rezolla and Zanotti (2013) Eq. 11.94(a)` in all the places you mention equations. The reason is that if someone in a year from now looks at line 247 they'll have to figure out what reference Eq. x.y is in, whereas if you always do `Rezolla and Zanotti (2013) Eq. x.y` it'll be much easier for them.